### ISSUE TITLE
Better integration with Valitron\Validator

### ISSUE THREAD

**Issue by nebulousGirl** [vlucas/spot2#125]
I had to use an event to add labels for my entity fields.

Also, there is a message hardcoded in English: FIELD is already taken.

We should be able to set our own Validator if there is the need, it would be far less limiting.

**Comment by jelofson** [comment]
Agreed. I did the same by using the beforeValidate event. I would be great to have access to the validator from the mapper. 
```
$validator = $mapper->getValidator();
$validator->labels(['field'=>'Label']);

$result = $mapper->update($entity);
```

Perhaps a way to set label/message via the 'validation' key on the field definition? Just a thought.
Overall, though, it works well.
